src/pb-restricted.js

```diff
@@ -51,8 +56,10 @@ export class PbRestricted extends pbMixin(LitElement) {
     render() {
         if (this.show) {
             this.style.display = '';
+            toggleFallback(this.fallback);
         } else {
             this.style.display = 'none';
+            toggleFallback(this.fallback);
         }
         return html`
             ${this.show && !this.disabled ? html`<slot></slot>` : null}
```

Comment:
If we could use a slot for the fallback, we could simply do

```javascript
return html`${this.show && !this.disabled ? html`<slot></slot>` : <slot name="fallback"></slot>}`;
```

Reply:
Oh yes, that would perfectly work, as well.